desc/vmec_utils.py

```diff
@@ -469,3 +486,575 @@ def vmec_boundary_subspace(eq, RBC=None, ZBS=None, RBS=None, ZBC=None):  # noqa:
     boundary_subspace = block_diag(Rb_subspace, Zb_subspace)
     opt_subspace = null_space(boundary_subspace)
     return opt_subspace
+
+
+def make_boozmn_output(  # noqa: 16 fxn too complex
```

Comment:
Should we move this into the `VMECIO` class? In my mind it is similar to the `VMECIO.save()` method since they are both outputting NetCDF files in a format that is specific to VMEC.

Reply:
And ideally we would rename them to something like `save_wout` and `save_boozmn`